### ISSUE TITLE
No wheels for python3.9

### ISSUE THREAD

**Issue by beneltayar** [scikit-fmm/scikit-fmm#64]
Hi,
There are no wheels for windows python3.9 on PyPI, even of the latest version, it would be nice to upload such a wheel 😃 
If you're lazy you can use the wheel I've built: [scikit_fmm_whl.zip](https://github.com/scikit-fmm/scikit-fmm/files/7475402/scikit_fmm_whl.zip)
Thanks!

**Comment by jkfurtney** [comment]
Hi Thanks for the note. I will work on getting a Python 3.9 AppVeyor build going.

**Comment by jkfurtney** [comment]
A while ago I tried adding a Python 3.9 wheel to AppVeyor but I ran into some problems. I had another quick look at this and it looks like a lot of open source projects are moving away from AppVeyor. We might be able to build Windows wheel files using github Actions. Does anyone know what the current best practice is for building Windows wheels for Python c extensions on a (free) CI system?

Here is what scikit-learn is doing with github workflows
https://github.com/scikit-learn/scikit-learn/blob/main/.github/workflows/wheels.yml
I will have a closer look at this soon.
